### PR TITLE
Remove explicit protocol mentions

### PR DESCRIPTION
**Pull request by haarts**
I'm endeavoring to understand the code base. Ultimately I would like to see if the backend (electrumX) could be loosely coupled with the rest.

In order to achieve this I'm hoping to push all explicit mentions of the protocol strings into the network module. This PR is a tiny step into that direction.

### CODE REVIEW

File: lib/commands.py [hunk 1]
```diff
@@ -647,8 +647,7 @@ def callback(x):
                 util.print_error('Got Response for %s' % address)
             except BaseException as e:
                 util.print_error(str(e))
-        h = self.network.addr_to_scripthash(address)
-        self.network.send([('blockchain.scripthash.subscribe', [h])], callback)
+        self.network.subscribe_to_addresses([addr], callback)
```
Comment (SomberNight) [inline lib/commands.py]: should be `address` not `addr`

File: lib/network.py [hunk 2]
```diff
@@ -641,13 +635,15 @@ def cb2(x):
         return cb2
 
     def subscribe_to_addresses(self, addresses, callback):
-        hashes = [self.addr_to_scripthash(addr) for addr in addresses]
-        msgs = [('blockchain.scripthash.subscribe', [x]) for x in hashes]
-        self.send(msgs, self.overload_cb(callback))
+        hash2address = {bitcoin.address_to_scripthash(address): address for address in addresses}
+        self.h2addr.update(hash2address)
+        msgs = [('blockchain.scripthash.subscribe', [x]) for x in hash2address.keys()]
```
Comment (SomberNight) [inline lib/network.py]: note: as a result of this, the subscriptions for `addresses` will be in a different than before order (sorted) on the wire, but I believe this is OK.

File: lib/commands.py (hunk 1 above)
Comment (haarts) [reply lib/commands.py]: will fix today

Comment (haarts) [reply lib/commands.py]: Ah. You already did. :) Thanks for catching that!

**Pull request merged**